platforms/extensibility/plugin-development/src/main/java/org/gradle/plugin/devel/plugins/JavaGradlePluginPlugin.java

```diff
@@ -94,7 +94,7 @@ public abstract class JavaGradlePluginPlugin implements Plugin<Project> {
 
     private static final Logger LOGGER = Logging.getLogger(JavaGradlePluginPlugin.class);
 
-    static final String API_CONFIGURATION = JvmConstants.API_CONFIGURATION_NAME;
```

Comment:
❔ Since it is visible in the same package and in sub-types, shouldn't the `API_CONFIGURATION` constant count as public API?  Even if no longer used, it should probably remain.